app/classes/Transvision/ShowResults.php

```diff
@@ -567,4 +581,286 @@ public static function buildErrorString($source_string, $target_string)
 
         return $error_message;
     }
+
+    /**
+     * Check if the locale has a project 'firefox' on Pootle.
+     *
+     * @param string $locale String the locale we want to check
+     *
+     * @return bool true if the locale is present is Pootle and has a project
+     *              'firefox', false otherwise.
+     */
+    public static function isPresentInPootle($locale)
+    {
+        static $localesInPootle = [
+            "ach",
```

Comment:
There's no need to define the variable as static inside a function, also use single quotes when double quotes are not needed.